pkg/cmd/cli/describe/describer.go

```diff
@@ -1328,3 +1311,48 @@ func (d *ClusterRoleBindingDescriber) Describe(namespace, name string) (string,
 	role, err := d.ClusterRoles().Get(roleBinding.RoleRef.Name)
 	return DescribeRoleBinding(authorizationapi.ToRoleBinding(roleBinding), authorizationapi.ToRole(role), err)
 }
+
+func describeBuildTriggerCauses(causes []buildapi.BuildTriggerCause, out *tabwriter.Writer) {
+	if causes == nil {
+		fmt.Fprintf(out, "\nBuild trigger cause <unknown>")
```

Comment:
This will print `Build trigger cause <unknown>`
but in case of ImageChange or other trigger, the output has an extra colon
`Build trigger cause:	Image change` 

Maybe use the `formatString()` to format the string ?